### PR TITLE
Add error messages

### PR DESCRIPTION
**Pull request by nattyait**
- Still don't write test cover the new code because it has to fake data to the right condition and I have no time now T_T
- Move message template to base.html so it's working for every template. (yeah it previously has the html template but in board.html so I cannot find it T_T)

Connected to #83.

### CODE REVIEW

File: zendesk_tickets_machine/boards/views.py [hunk 1]
```diff
@@ -244,7 +244,18 @@ def get(self, request, slug):
...
+                if ticket:
+                    each.zendesk_ticket_id = ticket.get('id')
+                else:
+                    result_description = result.get('description')
+                    result_details = result.get('details')
+                    items = result_details.popitem()
+                    for error_item in items[1]:
+                        item_description = error_item.get('description')
+                        item_error = error_item.get('error')
+                        error_text = result_description + ': ' + item_description + ' (' + item_error + ')'
```
Comment (zkan) [inline zendesk_tickets_machine/boards/views.py]: In Python 3.6+, we use string interpolation. https://www.python.org/dev/peps/pep-0498/

Also, I think this line breaks PEP8.

File: zendesk_tickets_machine/boards/views.py [hunk 2]
```diff
@@ -244,7 +244,18 @@ def get(self, request, slug):
...
+                        messages.error(request, error_text + "\n")
```
Comment (zkan) [inline zendesk_tickets_machine/boards/views.py]: Use string interpolation and single quote.

**Comment by nattyait** [comment]
I see the comment and will fix it when available na sorry T_T

**Comment by zkan** [comment]
@nattyait I've fixed this for you. The issue came quite often now. https://trello.com/c/sacf4skn/2235-bug-problem-ticket-machine-error-cb-monthly-tickets

**Pull request merged**